kiwi/defaults.py

```diff
@@ -470,15 +470,24 @@ def get_install_volume_id():
         return 'INSTALL'
 
     @staticmethod
-    def get_snapper_config_template_file():
+    def get_snapper_config_template_file(root: str) -> str:
         """
         Provides the default configuration template file for snapper
 
-        :return: file
+        :return: file path
 
         :rtype: str
         """
-        return '/etc/snapper/config-templates/default'
+        snapper_templates = [
+            'etc/snapper/config-templates/default',
+            'usr/share/snapper/config-templates/default'
+        ]
+        snapper_default_conf = ''
+        for snapper_template in snapper_templates:
+            snapper_default_conf = os.path.join(root, snapper_template)
+            if os.path.exists(snapper_default_conf):
+                break
+        return snapper_default_conf
```

Comment:
If neither of those files exists, it returns the last entry of the list instead of an empty string

Reply:
oh yes, true my bad, will fix it

Reply:
should be fixed now